io_scene_dae/export_dae.py

```diff
@@ -188,19 +188,15 @@ def export_image(self, image):
                 # folder
                 img_tmp_path = image.filepath
                 if img_tmp_path.lower().endswith(bpy.path.extensions_image):
-                    image.filepath =
-                        os.path.join(basedir, os.path.basename(img_tmp_path))
+                    image.filepath = os.path.join(basedir, os.path.basename(img_tmp_path))
```

Comment:
This reverts some changing done to ensure PEP-8 compliance. To be honest I prefer the one line form a thousand times, though I understand the need and advantages of following an established style guide.